index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>RESUME Roman Levsha</title>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="name">
+        <div class="container">
+          <div class="name__inf">
+          <h1>Levsha Roman</h1>
+          <p><em>JavaScript beginner programmer</em></</p>
+          </div>
+        </div>
+    </header>
+    <section class="basic">
+        <div class="container">
+          <div class="basic__inf">
+            <figure>
+              <img
+                src="/IMG_20200818_143736.jpg"
+                alt="Foto Roma"
+                width="190px">
+              <figcaption>R.Levsha.</figcaption>
+              </figure>
+            <section class="contacts">
+              <div class="contact">
+                <div class="contact__type">
+                  Phone(mts)
+                    <div class="contact__value">+375297635734</div>
+                </div>
+                <div class="contact__type">
+                  E-mail
+                    <div class="contact__value">rm.levsha@gmail.com</div>
+                </div>
+                <div class="contact__type">
+                  GitHub
+                    <div class="contact__value"><a href="https://github.com/levsha87">account github</a></div>
+                </div>
+              </div>
+            </section>
+        </div>
+      </div>
+    </section>
+    
+      <div class="main">
+        <div class="container">
+          <div class="main__inf">
+      <section class="my_goal">
+        <h2>My goal</h2>
+        <ol>
+          <li>
+            I want to get a new profession! I want to become high-quality
+            programmer.
+          </li>
+          <li>I want to raise living standards of my family.</li>
+          <li>
+            I want to study programming on JavaScript language to achieve my
+            goals.
+          </li>
+        </ol>
+      </section>
+      <section class="my_skills">
+        <h2>My skills</h2>
+        <p><b>Programming languages:</b></p>
```

Comment:
Using css styles instead of extra html-elements is considered as a good practice (<b> can be replaced with css-style). 
Plus, <h3> seems being here more suitable. As <p> is usually used for paragraphs, not titles.